tests/behat/features/bootstrap/EmailContext.php

```diff
@@ -71,10 +99,10 @@ public function getSpooledEmails() {
    *   Path to the file.
    *
    * @return string
-   *   An unserialized email.
+   *   An email.
    */
   public function getEmailContent($file) {
-    return unserialize(file_get_contents($file));
+    return file_get_contents($file);
```

Comment:
```suggestion
    return unserialize(file_get_contents($file), ["allowed_classes" => ["Drupal\symfony_mailer\Email", "Symfony\Component\Mime\Header\Headers", "Symfony\Component\Mime\Header\UnstructuredHeader", "Drupal\symfony_mailer\Address"]]);
```

The benefit of this is that we can change the return type to `\Drupal\symfony_mailer\Email` and we'll get unserialization errors if it's differently.

That'll help you downstream too.